.github/workflows/ci-infra.yml

```diff
@@ -64,3 +65,28 @@ jobs:
         uses: aquasecurity/tfsec-pr-commenter-action@v1.2.0
         with:
           github_token: ${{ github.token }}
+  # !! Uncomment to trigger automated infra tests once dev environment is set up
+  # infra-test-e2e:
```

Comment:
I kinda wonder if this should be in a separate file, so folks can choose whether they want to run E2E tests for every PR push vs. every commit to main. I imagine once projects become larger, folks may not want to run E2E tests every time a developer pushes up some new commits.

Not sure though!

Reply:
Hmm, good point. I'd still generally like for the tests to run before merge to reduce the chance that it breaks anything after merging, but I can see the value in not triggering them on every push/sync. I'll merge as is for now, seems like a reversible decision either way. One saving grace is that unlike `template-only-ci`, the e2e infra tests can actually run in parallel in a separate workspace, so even if a developer pushes up a commit, then immediately realizes they made a mistake and pushes up another commit, the second e2e test run can start without waiting for the previous one to finish